python-verso-routing/requirements.txt

```diff
@@ -0,0 +1,5 @@
+# Define the packages required by your project here.
+nextmv==0.27.0
+plotly==6.0.0
```

Comment:
If I am not mistaken, plotly can be dropped as a requirement. Or am I missing something?